zello-sdk/src/main/java/com/zello/sdk/Audio.java

```diff
@@ -72,18 +73,12 @@ void close() {
 	 * @return boolean indicating if the mode is available for the device.
      */
 	public boolean isModeAvailable(@NonNull AudioMode mode) {
-		switch (mode) {
-			case SPEAKER:
-				return _sp;
-			case EARPIECE:
-				return _ep;
-			case BLUETOOTH:
-				return _bt;
-			case WEARABLE:
-				return _wearables > 0;
-			default:
-				return false;
-		}
+		return switch (mode) {
+			case SPEAKER -> _sp;
+			case EARPIECE -> _ep;
+			case BLUETOOTH -> _bt;
+			case WEARABLE -> _wearables > 0;
```

Comment:
why keep if wearables are not supported anymore?

Reply:
I'm hesitant to just chop it off this time around. I think we need a separate ticket so we can set aside a little bit of time to consider what happens when an older version of the app that reports a wearable connects to the SDK.